webapp/src/Service/CheckConfigService.php

```diff
@@ -222,34 +222,42 @@ public function checkMysqlSettings(): ConfigCheckItem
         $desc = '';
         if ($vars['max_connections'] < 300) {
```

Comment:
I would write these to reduce duplication:
```
$desc .= sprintf("max_connections is set to %s.\n", $vars['max_connections']);
if ($vars['max_connections'] < 300) {
            $result = 'W';
            $desc .= sprintf("In our experience you need at least 300, but better 1000 connections to prevent connection refusal during the contest.\n", $vars['max_connections']);
}
```

Reply:
Changed.